Bottom: 3b84b969a7fce3782e7100385b8ba26665af72a1
Top:    143dee9002430f59aba90efc24b78e28476f4acd
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2008-09-30 01:16:09 +0100

Refresh of add-cache-for-noholes-diced-ve

---

diff --git a/src/mymem.c b/src/mymem.c
index 0352536..ffa4dad 100644
--- a/src/mymem.c
+++ b/src/mymem.c
@@ -727,7 +727,7 @@ FreePolygonMemory (PolygonTypePtr Polygon)
       MYFREE (Polygon->Points);
       if (Polygon->Clipped)
 	poly_Free (&Polygon->Clipped);
-      if (poly->NoHoles)
+      if (Polygon->NoHoles)
 	poly_Free (&Polygon->NoHoles);
       memset (Polygon, 0, sizeof (PolygonType));
     }
